src/test/java/com/iexec/worker/executor/TaskNotificationServiceTests.java

```diff
@@ -75,12 +77,10 @@ class TaskNotificationServiceTest {
     private TaskNotificationService taskNotificationService;
     @Captor
     private ArgumentCaptor<ReplicateStatusUpdate> replicateStatusUpdateCaptor;
+    @Mock
     private TaskDescription taskDescription;
 
-    @BeforeEach
     void init() {
```

Comment:
This doesn't seem to be an init function anymore. Should it be renamed, so that it is easier to know what it is actually doing? Something like `mockChainCalls`?

Reply:
Fixed in b379518